com.alovoa.alovoa-electron.yml

```diff
@@ -0,0 +1,81 @@
+app-id: com.alovoa.alovoa-electron
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
22.08 is the latest runtime.

Reply:
21.08 will be unmaintained in 6 months, so yeah, this needs updating to 22.08.